public/app/plugins/datasource/grafana-azure-monitor-datasource/components/MetricsQueryEditor/MetricsQueryEditor.test.tsx

```diff
@@ -109,46 +109,8 @@ describe('Azure Monitor QueryEditor', () => {
       azureMonitor: {
         ...mockQuery.azureMonitor,
         metricName: 'metric-b',
-      },
-    });
-  });
-
-  it('should auto select a default aggregation if none exists once a metric is selected', async () => {
```

Comment:
@sarahzinger this touches your first PR :sweat_smile: now we are removing the `aggregation` when changing the metric since it may not be available for that metric (and we don't have the list of aggregations at that moment).